jobs/JGDAS_GLOBAL_OCEAN_ANALYSIS_LETKF

```diff
@@ -12,9 +12,14 @@ GDATE=$(date --utc +%Y%m%d%H -d "${PDY} ${cyc} - ${assim_freq} hours")
 gPDY=${GDATE:0:8}
 gcyc=${GDATE:8:2}
 
+export GDUMP=${GDUMP:-"gdas"}
+export GDUMP_ENS="enkf${GDUMP}"
+
 YMD=${gPDY} HH=${gcyc} declare_from_tmpl -rx \
-   COM_OCEAN_HISTORY_PREV:COM_OCEAN_HISTORY_TMPL \
-   COM_ICE_HISTORY_PREV:COM_ICE_HISTORY_TMPL
+   COMIN_OCEAN_HISTORY_PREV:COM_OCEAN_HISTORY_TMPL \
+   COMIN_ICE_HISTORY_PREV:COM_ICE_HISTORY_TMPL
```

Comment:
If this is a job creating OCEAN ANALYSIS, why is COMIN_ICE_HISTORY required?

Reply:
The ocean and seaice DA is coupled @aerorahul . We need the backgrounds for both, ocean and seaice.

Reply:
ocean=ocean+wave+seaice+biogeochem+...

Reply:
Would this work `biogeochem=ocean+wave+seaice+biogeochem+...`?

Reply:
Since that would touch all the current ocean tasks, doesn't that constitute an issue beyond this PR?

Reply:
> Would this work `biogeochem=ocean+wave+seaice+biogeochem+...`?

No. The definition we set is Ocean DA: ocean+seaice+wave+biogeochem+ .... DA. 
So, when we need the atmospheric state for the ocean DA, will that be an issue as well?

Reply:
> One can have `COMIN_ATMOS` if atmosphere data is needed for ocean analysis. The comment was to highlight that the script/job is creating both ocean and ice analysis (if that is not true, then I retract the comment). If it is creating both ocean and ice analysis, then the j-job name should reflect that.

Would "marine= ocean+seaice+..." be acceptable?

Reply:
Rename the script to `JGDAS_GLOBAL_MARINE_ANALYSIS_LETKF`?

Reply:
With the understanding that the rest of the scripts and tasks will be similarly renamed?

Reply:
> With the understanding that the rest of the scripts and tasks will be similarly renamed?

yes

Reply:
The convention for the name of the j-job is `J<RUN>_ACTION` where `RUN` can be `gdas|gfs|enkfgdas|enkfgfs`.  If the same job should run for multiple `RUN` values such as `gdas|gfs`, then it should be named `JGLOBAL`

Reply:
What does `ocn` in the task names become? `marine`?

Reply:
Aside from the job name issues, I only see `COMIN`s here. Where is output going?

Reply:
That will be addressed when I add the finalize method. My understanding is that nothing should go in the PR that is not used immediately.

Reply:
So should `ocnanalletkf` be `marineanalletkf`?